view/library/browse.php

```diff
@@ -75,8 +78,11 @@
 					<?php
 					$sendToEditorUrl = KalturaHelpers::generateTabUrl( array( 'tab' => 'kaltura_browse', 'kaction' => 'sendtoeditor', 'entryIds' => array( $mediaEntry->id ) ) );
 					?>
-
-					<div id="entryId_<?php echo esc_attr( $mediaEntry->id ); ?>" class="showName" title="<?php echo esc_attr( 'Click to edit' ); ?>">
+					<?php
+					$canUserEditMedia = $kalturaModel->canLoggedInUserEditMedia($mediaEntry);
```

Comment:
You should init `$canUserEditMedia` in the controller and not in the view.
`$kalturaModel` should not be referenced here.